lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -12,12 +12,11 @@
 /**
  * Guaranteed context.querySelector. Always returns an element or throws if
  * nothing matches query.
- * @param {string} query
+ * @template {string} T
+ * @param {T} query
  * @param {ParentNode} context
- * @return {HTMLElement}
  */
 function find(query, context) {
```

Comment:
we apparently have four copies of `find()` :)

Reply:
soon to be 5 with #11832 

I often copy `Dom` in personal projects because the interface is so nice.